nym-api/src/nym_contract_cache/handlers.rs

```diff
@@ -17,59 +16,52 @@ use std::collections::HashSet;
 
 pub(crate) fn nym_contract_cache_routes() -> Router<AxumAppState> {
     Router::new()
+        .route("/mixnodes", axum::routing::get(get_mixnodes))
         .route(
-            &static_routes::v1::mixnodes(),
-            axum::routing::get(get_mixnodes),
-        )
-        .route(
-            &static_routes::v1::mixnodes::detailed(),
+            "/mixnodes/detailed",
             axum::routing::get(get_mixnodes_detailed),
         )
+        .route("/gateways", axum::routing::get(get_gateways))
+        .route("/mixnodes/rewarded", axum::routing::get(get_rewarded_set))
         .route(
-            &static_routes::v1::gateways(),
-            axum::routing::get(get_gateways),
-        )
-        .route(
-            &static_routes::v1::mixnodes::rewarded(),
-            axum::routing::get(get_rewarded_set),
-        )
-        .route(
-            &static_routes::v1::mixnodes::rewarded_detailed(),
+            "/mixnodes/rewarded/detailed",
             axum::routing::get(get_rewarded_set_detailed),
         )
+        .route("/mixnodes/active", axum::routing::get(get_active_set))
         .route(
-            &static_routes::v1::mixnodes::active(),
-            axum::routing::get(get_active_set),
-        )
-        .route(
-            &static_routes::v1::mixnodes::active_detailed(),
+            "/mixnodes/active/detailed",
             axum::routing::get(get_active_set_detailed),
         )
         .route(
-            &static_routes::v1::mixnodes::blacklisted(),
+            "/mixnodes/blacklisted",
             axum::routing::get(get_blacklisted_mixnodes),
         )
         .route(
-            &static_routes::v1::gateways::blacklisted(),
+            "/gateways/blacklisted",
             axum::routing::get(get_blacklisted_gateways),
         )
         .route(
-            &static_routes::v1::epoch::reward_params(),
+            "/epoch/reward_params",
             axum::routing::get(get_interval_reward_params),
         )
-        .route(
-            &static_routes::v1::epoch::current(),
-            axum::routing::get(get_current_epoch),
-        )
+        .route("/epoch/current", axum::routing::get(get_current_epoch))
 }
 
+#[utoipa::path(
+    tag = "contract-cache",
+    get,
+    path = "/v1/mixnodes",
+    responses(
+        (status = 200, body = Vec<MixNodeDetails>)
+    )
+)]
 async fn get_mixnodes(
     extract::State(state): extract::State<AxumAppState>,
 ) -> axum::Json<Vec<MixNodeDetails>> {
     state.nym_contract_cache().mixnodes_filtered().await.into()
 }
 
-// TODO dz:see deprecated warnings below
+// TODO dz:is this warning still relevant?
```

Comment:
eh. kinda. it's complicated. we can't remove it because people might be relying on it. I've explicitly deprecated all the routes in swagger on my directory v3 branch